cerbero/utils/shell.py

```diff
@@ -28,6 +28,7 @@
 import glob
 import shutil
 import hashlib
+import ntpath
```

Comment:
why do you use ntpath ? Shall we use everywhere ? ( ie _get_download_mirror_url)

Reply:
ntpath deals with '\' and also '/' correctly on all platforms. os.path should be enough here. I will change it.